app/main.py

```diff
@@ -1,3 +1,57 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from decimal import Decimal
+
+from app.customer import Customer, Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        json_data = json.load(file)
+    Car.change_fuel_price(json_data["FUEL_PRICE"])
```

Comment:
This is nice way to handle it. I guess you could also make it more simple if you make a separate module for getting json data and then pass this constant straight to class definition in Car, But you can keep it as is - it works